### PR TITLE
allow passing multiple arguments to dependency DSL calls

### PR DESCRIPTION
**Pull request by JT122406**

### CODE REVIEW

**Review by triphora — COMMENTED** [review]
I know this doesn't break API, but would it break ABI since the signature of the methods changed?

**Comment by JT122406** [comment]
> I know this doesn't break API, but would it break ABI since the signature of the methods changed?

It shouldn't

File: src/main/java/com/modrinth/minotaur/dependencies/container/NamedDependencyContainer.java [hunk 1]
```diff
@@ -27,19 +27,21 @@ protected NamedDependencyContainer(NamedDomainObjectContainer<NamedDependency> c
...
 	 * Creates an incompatible Dependency Container and applies the projectId property
 	 *
-	 * @param projectId the project id
+	 * @param projectIds the project id(s)
 	 */
-	public void project(final String projectId) {
-		this.dependencyContainer.add(new NamedDependency(projectId, null, this.dependencyType));
+	public void project(final String... projectIds) {
+		for (String projectId : projectIds)
+			this.dependencyContainer.add(new NamedDependency(projectId, null, this.dependencyType));
```
Comment (triphora) [inline src/main/java/com/modrinth/minotaur/dependencies/container/NamedDependencyContainer.java]:
```suggestion
		for (String projectId : projectIds) {
			this.dependencyContainer.add(new NamedDependency(projectId, null, this.dependencyType));
		}
```

File: src/main/java/com/modrinth/minotaur/dependencies/container/NamedDependencyContainer.java [hunk 2]
```diff
@@ -27,19 +27,21 @@ protected NamedDependencyContainer(NamedDomainObjectContainer<NamedDependency> c
...
 	 * Creates a incompatible Dependency Container and applies the versionId property
 	 *
-	 * @param versionId the version id
+	 * @param versionIds the version id(s)
 	 */
-	public void version(final String versionId) {
-		this.dependencyContainer.add(new NamedDependency(null, versionId, this.dependencyType));
+	public void version(final String... versionIds) {
+		for (String versionId : versionIds)
+			this.dependencyContainer.add(new NamedDependency(null, versionId, this.dependencyType));
```
Comment (triphora) [inline src/main/java/com/modrinth/minotaur/dependencies/container/NamedDependencyContainer.java]:
```suggestion
		for (String versionId : versionIds) {
			this.dependencyContainer.add(new NamedDependency(null, versionId, this.dependencyType));
		}
```

**Review by triphora — APPROVED** [review]
Very well, just some code formatting stuff

**Pull request merged**